Library/Formula/scalastyle.rb

```diff
@@ -2,24 +2,24 @@
 
 class Scalastyle < Formula
   homepage "http://www.scalastyle.org/command-line.html"
-  url "https://oss.sonatype.org/content/repositories/releases/org/scalastyle/scalastyle-batch_2.10/0.5.0/scalastyle-batch_2.10-0.5.0-distribution.zip"
-  version "0.5.0"
-  sha1 "bf009e0c4d48e5effce5113f5feb3da626ca1248"
+  url "https://oss.sonatype.org/content/repositories/releases/org/scalastyle/scalastyle_2.10/0.6.0/scalastyle_2.10-0.6.0-batch.jar"
+  version "0.6.0"
```

Comment:
You should remove this line, Homebrew already extract the version from the URL (this is why the build failed).